src/modules/mc_att_control/mc_att_control_params.c

```diff
@@ -403,7 +403,20 @@ PARAM_DEFINE_FLOAT(MC_ACRO_P_MAX, 120.0f);
  * @increment 5
  * @group Multicopter Attitude Control
  */
-PARAM_DEFINE_FLOAT(MC_ACRO_Y_MAX, 120.0f);
+PARAM_DEFINE_FLOAT(MC_ACRO_Y_MAX, 700.0f);
+
+/**
+ * Acro expo factor
```

Comment:
Can you describe that this applies to roll, pitch & yaw? And maybe that a linear combination is used between the 2 extremas?
Other than that, it's ready to be test-flown.